src/main/java/dataStructuresAndAlgorithms/challenges/multiBracketValidation/MultiBracketValidation.java

```diff
@@ -0,0 +1,41 @@
+package dataStructuresAndAlgorithms.challenges.multiBracketValidation;
+
+import dataStructuresAndAlgorithms.dataStructures.stacksAndQueues.Stack;
+
+public class MultiBracketValidation {
+
+    // This solution was influenced by a solution on The Polyglot Developer
+    public static boolean multiBracketValidation(String input) {
+        try {
+            if (input.length() % 2 != 0) { return false; }
```

Comment:
If I were to pass in the string `[m]` then the method will return false even though it should return true since the brackets match